src/util/address.rs

```diff
@@ -498,8 +498,8 @@ impl Address {
         }
     }
 
-    /// Create a pay to script address that embeds a witness pay to script hash address
-    /// This is a segwit address type that looks familiar (as p2sh) to legacy clients
+    /// Creates a pay to script address that embeds a witness pay to script hash address. This is a
+    /// segwit address type that looks familiar (as p2sh) to legacy clients.
```

Comment:
I feel like splitting a paragraph after the first phrase would be nicer here and in five instances above to shorten the summary.

Reply:
Sure thing, you mean adding an empty `///` right, so the paragraph is split in the rendered version? I'll do that.

Reply:
Yes, like this:

```rust
/// Creates a pay to script address that embeds a witness pay to script hash address.
///
/// This is a segwit address type that looks familiar (as p2sh) to legacy clients.
```

Reply:
Cool, I thought as much. Cheers